Revise interactions page (#171)
Pycord-Development/guide: docs/interactions/index.mdx

Text Modals are the most recently added Message Component in Discord. They're useful for text input and filling out forms, such as a 
sign-up for your bot's service. These are meant to replace long bot setup processes by allowing users to fill out multiple text fields, 
which avoids having the user send multiple messages for the bot. Modals can accessed by either invoking an Application Command or by 
interacting with another UI Component.

![Text Modal Example Image](../assets/interactions/modal-example.png)

This is what a Text Modal looks like, easy to use and one of the most useful Message Components yet. To learn more about Text Modals, 
please visit our [Modal Dialogs page](./ui-components/modal-dialogs).

## Application Commands

Application Commands are another set of new features that are intended to avoid compromising users' safety and privacy. 
They're relatively easy to add to your bot, and give people a simpler and safer way to use commands.

### [Slash Commands](https://docs.pycord.dev/en/master/api.html#slashcommand)

Slash Commands were the first Interaction added to Discord. They're easy to use and create, and

> allow for an alternative to prefix commands without needing the Message Content intent.

Reviewer: ```suggestion
is the only prefix commands alternative that does not need Message Content intent.
```

<br />

> This is what a Slash Command looks like. Not very different from a prefix command,

Reviewer: ```suggestion
This is what a Slash Command looks like. Not too different from a prefix command,
```
